content/blog/chif-driver-not-found.md

```diff
@@ -0,0 +1,121 @@
+---
+title: CHIF driver not found
+date: 2021-06-22T12:06:33.680Z
+author: François Donzé
+authorimage: https://gravatar.com/avatar/6f3982d1461459da475f47e3a6c89d1d?s=192
+tags:
+  - ilo-restful-api
+  - redfish
+  - chif driver
+  - ilorest
+  - ilo 5
+---
+## Introduction
+
+When using the HPE RESTful Interface Tool ([iLOrest](http://hpe.com/info/resttool)) or other applications, like the Smart Update Manager ([SUM](https://www.hpe.com/us/en/servers/smart-update.html)) or the integrated Smart Update Tool ([iSUT](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_US&docId=emr_na-a00068223en_us)), you may get errors similar to the one shown below without any other information to identify the origin of the problem or how to fix it.
+
+Chif driver not found, please check that the chif driver is installed
+
+This article attempts to explain what causes this error and why.
+
+## What is the CHIF driver ?
+
+The Channel Interface (CHIF) driver is an HPE proprietary driver allowing  in-band communication between applications and the Insight Lights Out (iLO) within HPE iLO based servers. It is stored as a dynamic loadable library file (`.dll`) on Microsoft Windows and a shared object (`.so`) on Linux.
```

Comment:
Here, it looks as though there is an extra space before "in-band", but in the preview it looks fine, so there may be no issue.